integrations/docker/images/chip-build-nrf-platform/Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG VERSION=latest
 
 FROM connectedhomeip/chip-build:${VERSION} as build
 # Compatible Nordic Connect SDK revision.
-ARG NCS_REVISION=v2.3.0
```

Comment:
Might be also a good idea to update zephyr-sdk to 0.16.0 and west to 1.0.0 below to match what our toolchain manager installs.

Reply:
Good point, done.